### ISSUE TITLE
Multiple relays

### ISSUE THREAD

**Issue by bajki** [pulsartronic/ESP8266Relays#1]
Hi can it be done to add multiple relays

**Comment by pulsartronic** [comment]
@bajki Yep, it is the idea, take a look at [ESP8266Relays.cpp](https://github.com/pulsartronic/ESP8266Relays/blob/master/ESP8266Relays/ESP8266Relays.cpp)

**Comment by bajki** [comment]
Nice. it works like a charm.

One more question,
it would be nice if it has client like vue so it can be wrapped in electron or cordova and run it on pc/phone.

**Comment by pulsartronic** [comment]
@bajki it actually has a client ... you can play with a demo here
http://pulsartronic.com/section/ESP8266RelaysApp/index.html

That's a demo ... and here is the actual client
http://pulsartronic.com/section/ESP8266RelaysApp/app.html

Use it from http, not https because, though it is possible, the esp8266 is not configured to use it now ...

Here is a tutorial, https://www.hackster.io/pulsartronic/programmable-wifi-digital-relay-switch-esp8266-app-code-6b5bf8

**Comment by bajki** [comment]
Can this be converted for ESP32?

**Comment by pulsartronic** [comment]
@bajki sure ... I never tried ... it shouldn't be difficult ... you would have to change references to ESP8266, for instance ESP8266WebServer and so on ...
